tmt/steps/execute/internal.py

```diff
@@ -327,8 +327,10 @@ def _run_tests(self, guest, extra_environment=None):
                     f"{duration} {test.name} [{progress}]", shift=shift)
                 if self._handle_reboot(test, guest):
                     continue
-            self._results.append(result)
             abort = self.check_abort_file(test)
+            if abort:
+                result.note = 'aborted'
```

Comment:
You mean modify `/tests/execute/restraint/tmt-abort` ? 
This is tested in `tests/execute/result/` but more tests don't hurt.

Reply:
Oh, good point. I though about checking that there is `note: aborted` inside `results.yaml` in `/tests/execute/restraint/tmt-abort` since this should check the correct functionality of `tmt-abort` but I think your addition to `tests/execute/result` is sufficient so feel free to ignore.

Reply:
I'll check for result and note in results.yaml. It is an easy change